pyproject.toml

```diff
@@ -30,3 +31,9 @@ select = ['E', 'W', 'F', 'C', 'N']
 ignore = ['E501']
 line-length = 95
 exclude = ["ca_cdk_constructs/eks/imports"]
+
+[[tool.mypy.overrides]]
```

Comment:
why the double square brackets?

Reply:
Good question, that was the example from mypy. Not sure what it's doing in pyproject syntax, but I'll have to have a look

Reply:
It's the [TOML table array syntax](https://toml.io/en/v1.0.0#array-of-tables). It means you can have multiple `[[tool.mypy.overrides]]` elements, each one setting a different config for different packages.